### PR TITLE
Fixing setup.py clean command

### PR DESCRIPTION
**Pull request by diraol**

### CODE REVIEW

**Comment by beraldoleal** [comment]
@diraol tests.

**Comment by diraol** [comment]
@beraldoleal fixed. It was a pylint error (error on pylint library)

File: setup.py [hunk 1]
```diff
@@ -5,6 +5,7 @@
 """
 import os
 from abc import abstractmethod
+from distutils.command.clean import clean  # pylint: disable=E0401,E0611
```
Comment (cemsbr) [inline setup.py]: Could you also add why this is being ignored (copy/paste from kytos/setup.py)? Future Us will be thankful :)

**Comment by diraol** [comment]
done @cemsbr ! =)

**Pull request merged**